README.md

```diff
@@ -26,6 +26,25 @@ $ celery -A origamid worker -l info
 $ origamid run_server
 ```
 
+### Installation with Docker :
+
+1. Clone the origami-daemon repository to your machine via git
+```
+git clone https://github.com/Cloud-CV/origami-daemon.git && cd origami-daemon
+```
+
+2. Then execute the following command: 
+```
+sudo docker-compose up
+```
+
+For easy calling , execute the following command:
+
+```
+chmod +x ./wrapper/origami-daemon
```

Comment:
I don't think we need a wrapper here, docker-compose is sufficiently easy for someone to start with the project.